recipes/desman/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: desman
+  version: "2.1"
+
+build:
+  number: 0
+  skip: True  # [py<35]
+
+extra:
+  skip-lints:
+    - setup_py_install_args
+
+source:
+  fn: DESMAN-v2.1.tar.gz
+  url: https://github.com/chrisquince/DESMAN/archive/v2.1.tar.gz
+  md5: 13d0669f4969777489e794fae4130bdd
+  sha1: 5856a971b2c870808fcbcbfdd732078326c05a5a
+  sha256: 3bca35dcabf1ce540ea053aaae459971b1ea9a1300d0b544002f50873d6347a2
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+    - gsl {{ CONDA_GSL }}*
+    - pandas
+    - scipy
+  run:
+    - python
+    - cython
+    - perl
+    - bcbiogff
+    - biopython
+    - pysam
+    - scipy
+    - numpy
+    - pandas
+    - scikit-learn
+    - gsl {{ CONDA_GSL }}*
+    - r-base 3.4.3
```

Comment:
R version is pinned by Bioconda in /scripts/env_matrix.yml so you can remove it here.